docs/compute-element/hosted-ce.md

```diff
@@ -46,6 +46,14 @@ Below is an example of the monitoring views that will be available for your clus
 
 ![monitoring graphs](/compute-element/img/monitoring_graphs.png)
 
+Each community is mapped to your site with a generic user account, with names osg01 - osg20. The mappings are as follows:
+
+| User | VO(s) | FQANs |
```

Comment:
```suggestion
| User | Community | FQANs |
```

We don't call them VOs in this doc for whatever reason but let's be consistent about it